pkgs/tools/security/yubikey-agent/yubikey-agent.service

```diff
@@ -0,0 +1,35 @@
+[Unit]
```

Comment:
This could also just end up being a template in https://github.com/FiloSottile/yubikey-agent.git. Wanna file a PR?

We might also want to add a `yubikey-agent.socket` file, to start the unit on demand when  `%t/yubikey-agent/yubikey-agent.sock` is accessed.

Reply:
I opened a PR at https://github.com/FiloSottile/yubikey-agent/pull/43.

Reply:
I'm quite new to systemd, but I'm not sure that `yubikey-agent.socket` will work for us?  yubikey-agent doesn't listen on stdin, it listens on a named file, which doesn't seem compatible with allowing systemd to pass the socket in to yubikey-agent?  I've probably misunderstood something though.

Reply:
Erp, I assumed this already was able to use sockets passed in from systemd (https://vincent.bernat.ch/en/blog/2018-systemd-golang-socket-activation). I'll remove the `.socket` bits from the PR, thanks!